File: Difficulty: Easy/Move All Zeroes to End/move-all-zeroes-to-end.cpp
class Solution {
  public:
    void pushZerosToEnd(vector<int>& arr) {
        // code here
        int n = arr.size();
        int zero=-1;
        for(int i = 0 ; i< n ; i++){
            if(arr[i]==0){
                zero=i;
                break;
            }
        }
        if(zero==-1)return;
        for(int i=zero+1;i< n ; i++){
            
            if(arr[i]!=0){
                swap(arr[i],arr[zero]);
                zero++;
            }
        }
    }
};

File: Difficulty: Easy/Move All Zeroes to End/move-all-zeroes-to-end.py
class Solution:
	def pushZerosToEnd(self, arr):
    	# code here
    	s = -1
    	n = len(arr)
    	for i in range(0, n):
    	    if arr[i] == 0:
    	        s = i
    	        break
    	if s==-1:return 
    	for i in range(s,n):
    	    if arr[i]!=0:
    	        arr[s],arr[i]=arr[i],arr[s]
    	        s+=1
    	        

